src/android/CustomFCMReceiverPlugin.java

```diff
@@ -1,44 +1,24 @@
 package org.apache.cordova.firebase;
 
 import android.os.Bundle;
-import android.os.Build;
-import android.text.TextUtils;
 import android.util.Log;
-import android.telecom.PhoneAccount;
-import android.telecom.PhoneAccountHandle;
-import android.telecom.TelecomManager;
 import android.content.ComponentName;
 import android.content.Intent;
 import android.content.Context;
-import android.content.pm.ApplicationInfo;
-import android.content.pm.PackageManager;
-import androidx.core.content.ContextCompat;
-import androidx.core.app.ActivityCompat;
-import android.app.Activity;
-import android.Manifest;
-
-import org.apache.cordova.CordovaInterface;
-import org.apache.cordova.CordovaPlugin;
-import org.apache.cordova.CordovaWebView;
+
+import com.dmarc.cordovacall.MyConnectionService; // TODO dereference by switching to implicit intent
```

Comment:
Working on removing this reference + switching to implicit intent....

Reply:
Change of plans: am going to intentionally postpone doing this - I think were headed in the direction of just using all plugins everywhere. We can always circle back and solve this if needed